### PR TITLE
Button changes are done.

### PR DESCRIPTION
**Pull request by MehmetOguzhanTor**
The "Add a New Habit" option is now "New Habit" with a big "+" on the button. And now this button is only red and does not change color because in the next features the chosen habit buttons will change their colors when they are achieved.

image

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -10,27 +10,35 @@ const ButtonComponent = ({onPress, title}) => {
     };
     const buttonStyle = {
         ...styles.button,
-        backgroundColor: isPressed ? 'red' : 'green',
```
Comment (ebzkurt1) [inline src/components/Button.js:13]: this is an if case, if you make the options same it serves no purpose. just write `'red'` here if that will be the only option

File: src/components/Button.js [hunk 2]
```diff
@@ -10,27 +10,35 @@ const ButtonComponent = ({onPress, title}) => {
...
         justifyContent: 'center',
         alignItems: 'center',
         backgroundColor: 'blue',
     },
     buttonText: {
         color: 'white',
         fontSize: 20,
+        textAlign: 'center',
+        marginTop: 10
+
```
Comment (ebzkurt1) [inline src/components/Button.js:41]: you can remove this empty line here

File: src/screens/HomeScreen.js [hunk 3]
```diff
@@ -13,31 +13,34 @@ const HomeScreen = ({ navigation, route }) => {
...
+              <Text style={styles.buttonTextStyle}></Text>
+              <Text style={styles.buttonTextStyle}>New Habit</Text>
+            </ButtonComponent>
+            {selectedHabit && (
         <Text style={styles.selectedHabitText}>
           Selected Habit: {route.params?.selectedHabit}
         </Text>
       )}
-    </View>
+        </View>
```
Comment (ebzkurt1) [inline src/screens/HomeScreen.js:26]: you can better indent these lines so that they look better in code

**Review by ebzkurt1 — CHANGES_REQUESTED** [review]
requested some changes

File: src/components/Button.js (hunk 1 above)
Comment (MehmetOguzhanTor) [reply src/components/Button.js:13]: Fixed.

File: src/screens/HomeScreen.js (hunk 3 above)
Comment (MehmetOguzhanTor) [reply src/screens/HomeScreen.js]: Indent is fixed.

File: src/components/Button.js (hunk 2 above)
Comment (MehmetOguzhanTor) [reply src/components/Button.js]: The empty line is removed.

**Pull request merged**